sdk/tests/js/tests/webgl_multiview_util.js

```diff
@@ -172,6 +172,39 @@ function getMultiviewColorFragmentShaderForDrawBuffers(drawBuffers) {
     return wtu.replaceParams(shaderCode, {'drawBuffers' : drawBuffers});
 }
 
+function getMultiviewVaryingVertexShader(views) {
+    let shaderCode = ['#version 300 es',
+    '#extension GL_OVR_multiview : require',
+
+    'layout(num_views = $(num_views)) in;',
+
+    'in vec4 a_position;',
+    'out float testVarying;',
+
+    'void main() {',
+    '    gl_Position = a_position;',
+    '    testVarying = float(gl_ViewID_OVR);',
+    '}'].join('\n');
+    return wtu.replaceParams(shaderCode, {'num_views': views});
+}
+
+function getMultiviewVaryingFragmentShader() {
+    return ['#version 300 es',
+    '#extension GL_OVR_multiview : require',
```

Comment:
My preference for formatting of these arrays is:
return [
    'a',
    'b',
].join('\n');

I don't mind this either way, but consider it!